pontoon/base/templates/translate.html

```diff
@@ -241,7 +253,8 @@ <h3 class="no-match"><div class="fa fa-exclamation-circle"></div>No results</h3>
     </nav>
 
     <textarea {% if not user.is_authenticated() %} readonly {% endif %} dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
-
+    <ul id="placeableMenu">
+    </ul>
```

Comment:
Should be in one line. And have a blank line before and after, like other siblings.